packages/app/src/cli/prompts/release.ts

```diff
@@ -18,17 +24,22 @@ export async function confirmReleasePrompt(
     })
   }
 
-  if (versionsDiff.removed.length > 0) {
+  // Filter out app config extensions in the prompt
+  const removed = versionsDiff.removed.filter(
+    (extension) => !isAppConfigSpecification(app, extension.specification.identifier),
+  )
```

Comment:
could this be a reusable helper given its identical to the "selector" above?

Reply:
✅ This duplicated code has been extracted into an `isNonAppConfigExtension` function via [VS Code's Extract Variable refactoring action](https://code.visualstudio.com/docs/editor/refactoring#_extract-variable) in 08be4612b08d933d47b544e962784e13def184e5.